app/views/DiscussionMessagesView.js

```diff
@@ -0,0 +1,216 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { FlatList } from 'react-native';
+import { useSelector } from 'react-redux';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import { HeaderBackButton } from '@react-navigation/stack';
+
+import ActivityIndicator from '../containers/ActivityIndicator';
+import I18n from '../i18n';
+import StatusBar from '../containers/StatusBar';
+import log from '../utils/log';
+import debounce from '../utils/debounce';
+import { themes } from '../constants/colors';
+import SafeAreaView from '../containers/SafeAreaView';
+import * as HeaderButton from '../containers/HeaderButton';
+import * as List from '../containers/List';
+import BackgroundContainer from '../containers/BackgroundContainer';
+import { isIOS } from '../utils/deviceInfo';
+import { getHeaderTitlePosition } from '../containers/Header';
+
+import { useTheme } from '../theme';
+import Message from '../containers/message';
+import RocketChat from '../lib/rocketchat';
+import SearchHeader from '../containers/SearchHeader';
+
+const API_FETCH_COUNT = 25;
+
+const DiscussionMessagesView = ({ navigation, route }) => {
```

Comment:
Can we rename it to `DiscussionsView`?
We talked about it Yesterday.
The idea is to rename `ThreadMessagesView` -> `ThreadsView` as well in the future 🤫